data/ships/ac33.json

```diff
@@ -6,17 +6,17 @@
 	"ship_class": "medium_freighter",
 	"min_crew": 3,
 	"max_crew": 5,
-	"price": 1015644,
+	"price": 420475,
 	"hull_mass": 780,
 	"atmospheric_pressure_limit": 7.9,
-	"capacity": 920,
+	"capacity": 609,
 	"slots": {
 		"engine": 1,
 		"cabin": 50,
 		"laser_front": 1,
 		"laser_rear": 1,
 		"missile": 26,
-		"cargo": 480
+		"cargo": 169
```

Comment:
I appreciate your attempt to provide a ship "equilibrium" here, but these changes to ship json will not be merged for two reasons: a) as discussed above, ships have more of a sidegrade path than a straight, evenly spaced upgrade path; and b) these values were recently rebalanced according to the physical capacity of the ship model and a system of equations that calculates the price of the ship from many different factors rather than just their utility in trade gameplay.
I'd certainly be interested in improving some of those equations related to cargo capacity in the future, but changing these values in a vacuum isn't desired at the moment.

Reply:
I will repeat my reply about price of commodities altered to fit this.
Cargo and capacities on existing ships come to give the feeling of progress during game. Of course i could not create my own chain of ships to demonstrate a working example of what an exponential in 'rn' can offer. So, i used existing ships to create a chain of them that increase their cargo in progress. These alterations are irrelevant to the exponential and so, we can safely remove this part and reject alterations on cargo and capacities without harming the scope of present PR. Doing so, ships, from real game play view, will be too expensive or too cheap. Of course, as i mention previously, i tried the game adding only the exponential. Exponential works as expected, but ships were out of place. Many were too expensive, initial ones were too cheap.
And i will end with the same phrase.

Alteration of ship json files is not important in this PR. They just demonstrate a working example of the consequences of the existence of an 'rn's exponential. In that scope they can safely rejected.